Makefile

```diff
@@ -29,14 +29,17 @@ LD_FLAGS_V2=" \
 IMAGE_NAME?=mpioperator/mpi-operator
 KUBEBUILDER_ASSETS_PATH := $(dir $(abspath $(firstword $(MAKEFILE_LIST))))bin/kubebuilder/bin
 KIND_VERSION=v0.17.0
+HELM_VERSION=v3.11.2
 # This kubectl version supports -k for kustomization.
 KUBECTL_VERSION=v1.25.6
 ENVTEST_K8S_VERSION=1.25.0
 PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
 GOARCH=$(shell go env GOARCH)
 GOOS=$(shell go env GOOS)
-# Use go.mod go version as a single source of truth of scheduler-plugins version.
-SCHEDULER_PLUGINS_VERSION?=$(shell awk '/scheduler-plugins/{print $$2}' go.mod|head -n1)
+# In the specific scheudler-plugins version such as v0.25.7, manifests are incorrect.
+# So we use the release branch for manifests.
+# Defaults to release-1.25 branch.
+SCHEDULER_PLUGINS_VERSION ?= v0.25.8-0.20230328191936-396d9098edd4
```

Comment:
This issue was fixed in the release branch: https://github.com/kubernetes-sigs/scheduler-plugins/pull/542.
However, the scheduler-plugins team didn't cut a new release.

Reply:
@HuangWei, do you mind cutting a release?

Reply:
@Huang-Wei (I put the wrong github username before)

Reply:
For now, it might be better to install manifests using `v0.25.7` (hold incorrect manifests) in the following instead of using untagged revision (v0.25.8-0.20230328191936-396d9098edd4):

```shell
helm install scheduler-plugins ./as-a-second-scheduler --create-namespace --namespace scheduler-plugins --set scheduler.image=registry.k8s.io/scheduler-plugins/kube-scheduler:v0.25.7 --set controller.image=registry.k8s.io/scheduler-plugins/controller:v0.25.7
```

This means we overwrite image versions when installing the scheduler-plugins.

Reply:
Done.